docs/sources/http_api/data_source.md

```diff
@@ -227,7 +332,10 @@ Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
   "database":"",
   "basicAuth":true,
   "basicAuthUser":"basicuser",
-  "basicAuthPassword":"basicuser",
+  "basicAuthPassword":"",
```

Comment:
Should `.basicAuthpassword` be part of the request?

Reply:
correct it should be omitted.